src/contracts/PreAMMBatcher.sol

```diff
@@ -102,10 +104,22 @@ contract PreAMMBatcher {
         return Fraction(f.denominator, f.numerator);
     }
 
+    function orderIsCurrentlyValid(Order memory order)
+        private
+        view
+        returns (bool)
+    {
+        return
+            // solhint-disable-next-line not-rely-on-time
+            (order.validUntil >= block.timestamp) &&
+            // solhint-disable-next-line not-rely-on-time
+            (order.validFrom <= block.timestamp);
+    }
```

Comment:
ubernit: I would disable and enable for the block, this way we don't have to worry about auto-formatting causing lint errors to appear out of nowhere:
```suggestion
    {
        // solhint-disable not-rely-on-time
        return
            (order.validUntil >= block.timestamp) &&
            (order.validFrom <= block.timestamp);
        // solhint-enable not-rely-on-time
    }
```

So, what I mean by this is what if the line length gets extended and the auto-formatter decides to format as:
```solidity
        return (order.validUntil >= block.timestamp) && (order.validFrom <= block.timestamp);
        // solhint-disable-next-line not-rely-on-time
        // solhint-disable-next-line not-rely-on-time     
```

Reply:
Very good point!